custom-recipes/nlp-google-cloud-translation-translate/recipe.json

```
@@ -939,6 +939,24 @@
       ],
       "defaultValue": "en"
     },
+    {
+      "name": "format",
+      "label": "Input Format",
```

Nitpick: harmonize case
```suggestion
      "label": "Input format",
```